test_util/EclRegressionTest.hpp

```diff
@@ -170,6 +170,7 @@ class ECLRegressionTest: public ECLFilesComparator {
                                                        };
     // keywords that should not be compared
     const std::vector<std::string> keywordsBlackList = {"TCPU", "ELAPSED", "TCPUDAY", "TCPUTS", "TELAPLIN", "TCPUH", "TCPUHT", "TCPUSCH", "TCPUTSH", "TCPUTSHT", "TELAPDAY", "TELAPTS"};
+    const std::vector<std::string> keywordsBlackListExtraRestart = {"MLINEARS", "MSUMLINS", "MSUMNEWT", "NEWTON", "NLINEARS", "NLINSMAX", "NLINSMIN"};
```

Comment:
I assume this is blacklisted because they are reset to 0 for a restart run. Is that the behaviour we want for the simulator? If it is, I think this can be merged. I could not see any flaws in the code.